app/backend/app.py

```diff
@@ -682,21 +668,6 @@ async def setup_clients():
             query_speller=AZURE_SEARCH_QUERY_SPELLER,
         )
 
-    current_app.config[CONFIG_CHAT_APPROACH] = ChatReadRetrieveReadApproach(
-        search_clients=search_clients,
```

Comment:
The multiple search clients that are build per usecase are used to change the chat approach. Don't think this will continue to work when we remove this